main/src/ca/uwaterloo/flix/language/fmt/FormatType.scala

```diff
@@ -86,6 +86,18 @@ object FormatType {
     formatTypeWithOptions(tpe, fmt)
   }
 
+  /**
+    * Transforms the given simple type into a string.
+    */
+  def formatSimpleType(tpe: SimpleType)(implicit flix: Flix): String =
+    formatSimpleTypeWithOptions(tpe, flix.getFormatOptions)
+
+  /**
+    * Transforms the given simple type into a string, using the given format options.
+    */
+  def formatSimpleTypeWithOptions(tpe: SimpleType, fmt: FormatOptions): String =
+    format(tpe)(fmt)
+
```

Comment:
This looks a bit boiler-platy. What's happening here? One helper that forwards to another? Perhaps at least formatSimplyTypeWithOptions can be moved into formatSimpleType?

@mlutze Any thoughts?

Reply:
It's just an attempt to somewhat mirror the rest of the API